### DISCUSSION TITLE
How to migrate from Uservoice to Fider?

### DISCUSSION THREAD

**Discussion by tobiasschuerg** [getfider/fider#1043] [Q&A] [answered]
I'd to migrate one forum with all ideas and its users to fider. Is that possible without much overhead?
Ideally just directly import the export from uservoice...?

**Comment by goenning** [comment, marked as answer]
I'm not aware of any migration script for that. Unless someone in the community has done that before, your best option would be to write a script and use Fider's API.

If you or anyone else end up doing it, please share here 🙏

**Comment by Cristy94** [comment]
Uservoice free plan is ending this year, so I see a lot of people migrating. It would be nice to have an easy export/import functionality.

**Reply by mattwoberts** [reply]
This is a very good point. I spotted you were looking at this recently @Cristy94, how did you manage to do it?

**Reply by Cristy94** [reply]
I only had around 30 ideas/suggestions, so I simply copy-pasted the contents of the suggestions to Fider, and added a screenshot from the old UserVoice ticket. Something like: https://feedback.uxwizz.com/posts/2/order-by-most-visited-sessions
